test/data/apple_watch_route.gpx

```diff
@@ -0,0 +1,1867 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<gpx version="1.1" creator="Apple Health Export" xmlns="http://www.topografix.com/GPX/1/1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.topografix.com/GPX/1/1 http://www.topografix.com/GPX/1/1/gpx.xsd">
+  <metadata>
+    <time>2019-12-19T04:53:04Z</time>
+  </metadata>
+  <trk>
+    <name>Route 2019-12-15 3:50pm</name>
+    <trkseg>
+      <trkpt lon="-81.097216" lat="32.064999"><ele>14.601502</ele><time>2019-12-15T20:19:55Z</time><extensions><speed>1.310662</speed><course>101.747269</course><hAcc>1.801682</hAcc><vAcc>1.189247</vAcc></extensions></trkpt>
```

Comment:
Could you truncate this file a bit so that the fixtures are smaller?

Reply:
I could, how long would you prefer it to be?

Reply:
The first 50 lines of trkpt's should be enough to cover the new functionality - thanks!